src/coprocessor/dag/aggr_fn/util.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::convert::TryFrom;
+
+use cop_datatype::builder::FieldTypeBuilder;
+use cop_datatype::{EvalType, FieldTypeAccessor, FieldTypeTp};
+use tipb::expression::{Expr, FieldType};
+
+use crate::coprocessor::dag::rpn_expr::impl_cast::get_cast_fn;
+use crate::coprocessor::dag::rpn_expr::types::RpnExpressionNode;
+use crate::coprocessor::dag::rpn_expr::{RpnExpression, RpnExpressionBuilder};
+use crate::coprocessor::Result;
+
+/// Checks whether or not there is only one child and the data type is supported.
```

Comment:
```suggestion
/// Checks whether or not there is only one child and the child expression is also supported.
```
The child can be a function too

Reply:
Changed to ".. and the return data type of the child is supported"

Reply:
Is it only about the return data type? I also see checks whether RPN functions are supported inside.